output-guidelines/micropublications.md

```diff
@@ -1,7 +1,70 @@
+  - [Rajan lab comics](https://www.rajanlab.com/comics)
+  - [Dance your PhD](https://www.youtube.com/results?search_query=dance+your+phd)
+
+### Author Contributions
+We request all the teams to track author contributions using the [CRediT taxonomy](https://credit.niso.org). Click on each of the 14 contributor roles to see what they constitute. We discourage giving credits to a scholar (including any former TAs) or mentor if they haven't contributed to the project at all. 
+- Format: .csv, .ods, or .tsv
+- Templates: We are providing 2 templates to track author contributions throughout the program. **Each team is required to submit author contributions in one of these 2 formats.** Download one of the following templates for your team:
+  - [CISP_contributors_table_template](https://docs.google.com/spreadsheets/d/140bNO83-GkOP1SYXWLpZmY27a5wxtj97gOR-5CaMXDQ/edit?usp=sharing): This template was adopted from [tenzing.club](https://contributorshipcollaboration.github.io/projects/tenzingclub). Feel free to leave blanks for Roles that aren't relevant for your team. Please use the Notes column to let us know of any specifics that aren't captured in the 14 categories (such a equal contribution, or additional roles other than the 14 that are a part of CRediT).
+  - [CISP_contributors_table_template_granular](https://docs.google.com/spreadsheets/d/1fw34ZjKaZQ9iv6ld6YT1kZP2uyg0CbmJIZdM88tb07E/edit?usp=sharing): This is a replica of the previous template with more granularity added for each role. There are 2 types of contributions to choose from:
+    - Lead: If a scholar has made major contributions in a role, choose this option.
+    - Support: If a scholar has primarily provided support and feedback or helped with brainstorming but hasn't made major contributions to the role, choose this option.
+   
+### Main Text
+Based on the type of research, please elaborate on the single claim, observation, or method. Think of this section as the combination of the Methods, Results, and Discussion sections of a journal-style research article. 
+  - Word limit: 1500 words
+  - Format: markdown (i.e. a .md or .markdown file). Please refer to [this official guide](https://daringfireball.net/projects/markdown/) for details on this open format.
+  - To make the results reproducible, authors may add supplementary methods if the 1500 word limit isn't sufficient to report the steps to reproduce the results.
+  - References : Please add external links to any references directly in the main text using the markdown format of [inline-style links](https://daringfireball.net/projects/markdown/syntax#link) where the link text in square brackets is immediately followed by the link in parentheses. Wherever possible please add DOIs for the citations. 
+      
+### Figure
+Support your claim, observation, or method with **a single figure**.
+  - Format: .svg or .png
+  - Add figure legends for each of the panels within this single figure.
```

Comment:
I am confused by this. Are we asking the scholars to have legend text on the image? That could be a headache to make and the result would not be searchable or machine-translatable. I think it would be better to include the legend text in the main text file or submit it as a separate markdown file.

Reply:
Good catch! Added a separate field for figure legend (which is just a long text field instead of another markdown file.